client/sites/settings/index.tsx

```diff
@@ -0,0 +1,69 @@
+import page from '@automattic/calypso-router';
+import { makeLayout, render as clientRender } from 'calypso/controller';
+import { siteSelection, sites, navigation } from 'calypso/my-sites/controller';
+import {
+	SETTINGS_SITE,
+	SETTINGS_ADMINISTRATION,
+	SETTINGS_AGENCY,
+	SETTINGS_CACHES,
+	SETTINGS_WEB_SERVER,
+} from 'calypso/sites/components/site-preview-pane/constants';
+import { siteDashboard } from 'calypso/sites/controller';
+import {
+	siteSettings,
+	administrationSettings,
+	agencySettings,
+	cachesSettings,
+	webServerSettings,
+} from './controller';
+
+export default function () {
+	page( '/sites/settings', siteSelection, sites, makeLayout, clientRender );
+	page(
+		'/sites/settings/site/:site',
+		siteSelection,
+		navigation,
+		siteSettings,
+		siteDashboard( SETTINGS_SITE ),
+		makeLayout,
+		clientRender
+	);
+
+	page(
```

Comment:
We need a site-selection route for each individual route listed here, similar to the first one above; for example for this it would be 

```
page( '/sites/settings/administration/:site', siteSelection, sites, makeLayout, clientRender );
```